### PR TITLE
UITEN-247: Improve "Number of locations" on Institutions page

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
## Purpose

[UITEN-247](https://issues.folio.org/browse/UITEN-247)
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by UladzislauKutarkin** [comment]
https://github.com/folio-org/ui-tenant-settings/assets/72550466/bfeb4b2f-39f9-419b-b557-e7f83a1f5be0

File: translations/ui-tenant-settings/en.json [hunk 1]
```diff
@@ -193,5 +193,6 @@
   "permission.settings.sso": "Settings (tenant): Can maintain SSO settings",
   "permission.settings.location": "Settings (tenant): Can create, edit and remove locations",
   "permission.settings.servicepoints": "Settings (tenant):  Can create, edit and remove service points",
-  "permission.settings.bursar-exports": "Settings (tenant): Bursar admin"
+  "permission.settings.bursar-exports": "Settings (tenant): Bursar admin",
+  "settings.location.institutions.number": "{number}"
```
Comment (artem-blazhko) [inline translations/ui-tenant-settings/en.json:197]: Do we really need it here?
We have only the "number" parameter without any word(s) that can be translated. Looks like it is redundant.

File: src/settings/LocationInstitutions.js [hunk 2]
```diff
@@ -63,15 +74,34 @@ class LocationInstitutions extends React.Component {
    * will be stale if they change between unmounting/remounting.
    */
   componentDidMount() {
-    this.props.mutator.locationsPerInstitution.reset();
```
Comment (artem-blazhko) [inline src/settings/LocationInstitutions.js:66]: Should we remove `locationsPerInstitution` from mutator object on line 23?

Comment (UladzislauKutarkin) [reply src/settings/LocationInstitutions.js:66]: Thanks! I missed that. Already fixed

File: translations/ui-tenant-settings/en.json (hunk 1 above)
Comment (UladzislauKutarkin) [reply translations/ui-tenant-settings/en.json]: Agree, I removed that

File: src/settings/LocationInstitutions.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/settings/LocationInstitutions.js:66]: @artem-blazhko please take a look at fixed changes

**Review by vashjs — APPROVED** [review]

File: src/settings/LocationInstitutions.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.numberOfObjectsWrapper {
+    color: #2f609f;
+    cursor: pointer;
+}
```
Comment (NikitaSedyx) [inline src/settings/LocationInstitutions.css:4]: EOF please

**Review by NikitaSedyx — APPROVED** [review]

File: src/settings/LocationInstitutions.css (hunk 3 above)
Comment (UladzislauKutarkin) [reply src/settings/LocationInstitutions.css]: Done

**Pull request merged**